App/Screens/Home/Footer/ShareButton/ShareImage/ShareImage.tsx

```diff
@@ -42,6 +42,7 @@ const styles = StyleSheet.create({
 export function ShareImage(): React.ReactElement {
 	const { api } = useContext(ApiContext);
 	const { currentLocation } = useContext(CurrentLocationContext);
+	const { t } = useTranslation()
```

Comment:
See line 58-64

I don't understand how the `t` function will be handled after passed to `CigarettesBlock` function. The code I traced down has been compressed pretty much so I don't think I could understand it. Could you help me to understand this case @amaurymartiny

Reply:
It's a way for me to pass down the `t` function down to the `@shootismoke/ui` package. Here's how it looks in that component: https://github.com/shootismoke/common/blob/master/packages/ui/src/CigarettesText/CigarettesText.tsx.

Does `t={t}` not work?

Reply:
I believe some translations in that `@shootismoke/ui` package need to be updated too. I created a PR https://github.com/shootismoke/common/pull/306, let me know if that PR helps.